### DISCUSSION TITLE
Machine Learning Modules in Kibana

### DISCUSSION THREAD

**Discussion by oneCrazyAdmin** [Security-Onion-Solutions/securityonion#3976] [General]
Is there any possibility of our SO installation going haywire by installing any of the other Elastic modules like the Machine Learning, or the SEIM?

**Comment by weslambert** [comment]
I wouldn't necessarily say that your installation would go haywire, but these components may not work as expected, or may not stay enabled throughout upgrade, as we've not heavily tested these features, so I would do so at your own risk.
